modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -419,8 +418,8 @@ sda.enkf.multisite <- function(settings,
         
         #I'm rewrting the runs because when I use the parallel appraoch for wrting configs the run.txt will get messed up; because multiple cores want to write on it at the same time.
         runs.tmp <- list.dirs(rundir, full.names = F)
+        runs.tmp <- runs.tmp[grepl("ENS-*", runs.tmp)]
```

Comment:
Not sure about this change. Depending on how the pecan.xml and bety are set-up you I'm not sure you can depend on this naming of ensemble members. Ultimately, a parrallel-safe approach to writing the runs list would be a better solution (e.g. data.frame::fwrite with append=TRUE looks promising)